src/algebra/quaternion.lean

```diff
@@ -696,6 +696,30 @@ instance : no_zero_divisors ℍ[R] :=
 instance : is_domain ℍ[R] :=
 no_zero_divisors.to_is_domain _
 
+/-- The only quaternions equal to their conjugates are real. -/
+lemma conj_eq_self_iff : conj a = a ↔ a ∈ set.range (coe : R → ℍ[R]) :=
```

Comment:
No, but I could add it. Which version do you think these lemmas should be stated with?

Down the line it would be nice to unify this with the `self_adjoint`/`skew_adjoint` stuff, but I don't want to do that mid-port.

Reply:
I think `a ∈ set.range (coe : R → ℍ[R])` is nicer and shorter. But indeed `self_adjoint`/`shew_adjoint` would be the superior spelling.

Reply:
Looking through the rest of the file I think `a = a.re` is the preferred spelling.

Reply:
Changed to use that spelling; I found that some of the lemmas I wanted already existed.